android-components/components/compose/cfr/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,7 @@
 <resources>
     <!-- Content description (not visible, for screen readers etc.): Description for the close button of a Contextual Feature Recommendation Popup. -->
     <string name="mozac_cfr_dismiss_button_content_description">Dismiss</string>
+
+    <!-- Content description: Text for a11y to be dictated when a CFR is displayed -->
+    <string name="mozac_cfr_a11y_notification_title">notification</string>
```

Comment:
let's make this a parameterized string, and the parameter can be the app name you obtain with `Context.appName`